node/src/chaincodeevents.test.ts

```diff
@@ -131,6 +132,145 @@ describe('Chaincode Events', () => {
             expect(startPosition?.getSpecified()?.getNumber()).toBe(Number(startBlock));
         });
 
+        it('Sends valid request with specified start block number and fresh checkpointer', async () => {
```

Comment:
There is a lot of cut & paste duplication in these tests. It would be good to reduce duplication either with some helper methods to repetitive work, or rewrite these as table tests, so the same test code runs with just different combinations of inputs and expected outputs